doc/plugins.md

```diff
@@ -201,6 +201,8 @@ See [Creating plugins][create] below.
     — custom syntax Wordpress- and Hugo-like shortcodes (new node type)
 *   [`remark-simple-plantuml`](https://github.com/akebifiky/remark-simple-plantuml)
     — convert PlantUML code to images
+*   [`remark-slate`](https://github.com/hanford/remark-slate)
+    — Serialize Markdown as a slate 0.50+ compatible object.
```

Comment:
```suggestion
    — compile Markdown to Slate [nodes](https://docs.slatejs.org/concepts/02-nodes)
```

How about this?
- “Compile” because [unified@9](https://github.com/unifiedjs/unified/releases/tag/9.0.0) uses that term for the generic action, “serialize” is used for text
- Dropped the version number, as I’m assuming you’ll update the project when they change in the future?
- Added a link!